analyzers/its/expected/Nancy/S3900-Nancy-netstandard2.0.json

```diff
@@ -1380,18 +1374,6 @@
       "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Nancy/src/Nancy/Routing/Trie/Nodes/TrieNode.cs#L192",
       "Location": "Line 192 Position 36-44"
     },
-    {
-      "Id": "S3900",
-      "Message": "Refactor this method to add validation of parameter \u0027localCaptures\u0027 before using it.",
-      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Nancy/src/Nancy/Routing/Trie/Nodes/TrieNode.cs#L224",
```

Comment:
[Link](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Nancy/src/Nancy/Routing/Trie/Nodes/TrieNode.cs#L224)
`NonNull` learnt in `localCaptures.Any()` on the `if` condition.